RevenueCat/Scripts/revenuecat.purchases-unity.asmdef

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Do we need these asmdef both here and in RevenueCat/Editor?

Reply:
Hmm well, this one is needed so we can access these files in projects using purchases-unity as a package. For example, for Subtester, without this, it fails with a bunch of errors like this: `Assets/Scripts/PurchasesListener.cs(8,34): error CS0246: The type or namespace name 'Purchases' could not be found (are you missing a using directive or an assembly reference?)`.

About the Editor one, I believe it would be needed if it needs to be accessed in projects. Honestly, not sure if it's needed but I added it just in case. Do you think we should remove it?

Reply:
I think it's good. I am actually not entirely sure what these files do lol